### ISSUE TITLE
[Fleet Server] Add dashboard for Fleet Server

### ISSUE THREAD

**Issue by mostlyjason** [elastic/integrations#812]
Let's add a dashboard for Fleet Server operators to help them scale the server when needed and troubleshoot issues

It should show:
- [ ] System metrics like CPU, memory usage by agent over time. Should be accurate for VMs and containers.
- [ ] Fleet server process metrics like CPU, memory usage by agent over time. Should be accurate for VMs and containers
- [ ] Status by agent over time
- [ ] Log stream component showing errors

Stretch:
- [ ] Number of active connections by agent over time
- [ ] Number of rejected connections by agent over time

Related issues:
- Logs, metrics and status info for Fleet Server https://github.com/elastic/beats/issues/24415

**Comment by simitt** [comment]
For which version is this planned?

**Comment by ph** [comment]
@simitt We haven't added it to the roadmap yet.

**Comment by mostlyjason** [comment]
@ravikesarwani Can I get your input on the Fleet Server dashboard definition? In particular, the open question about how it relates to the Elastic Agent dashboard?

**Comment by ravikesarwani** [comment]
@mostlyjason Yes, I would love to help and add my thoughts but don't have a lot of context around the topic of fleet server. Can we setup some time next week to discuss so that you can fill me in and then I can research and add my thoughts?

**Comment by michalpristas** [comment]
added a 7.14 label, please update if needed for another version

**Comment by EricDavisX** [comment]
@mostlyjason just pinging - I'm doing follow-ups on open issue for 7.14 - is this still under review / possible merge for 7.14?  Assuming not, I'll remove the 7.14 label, please add back if any work was done for this cycle.  If this is urgent for the Fleet / Agent GA we may yet have time to finish it out.

**Comment by nimarezainia** [comment]
@jlind23 i added this to the data plane board as for some reason GH is not finding the control plane board in this repo. Do you have the same issue?

**Comment by jlind23** [comment]
@nimarezainia Nope i don't have this issue. I just added it to the control plane board.

**Comment by jlind23** [comment]
This one is about exporting this newly created dashboard and pushing it to fleet-server package.

**Comment by jlind23** [comment]
@nimarezainia could you jump in here please?
